containers/init/clone/clone.sh

```diff
@@ -14,8 +14,11 @@
 # limitations under the License.
 
 set -ex
-
 cd /src/workspace
 ls -A | xargs -r rm -fr
-git clone --recursive $CLONE_REPO .
-git checkout $CLONE_GIT_REF
+git init
```

Comment:
nit: leave a comment that this seemingly weird way of cloning the repo is done for a good reason (and describe the problem it's avoiding).